documents/Final Report/evaluation.tex

```diff
@@ -3,20 +3,24 @@ \chapter{Evaluation and failure analysis}
 
 \section{Evaluation of the connector}
 The connector is pretty much where it needs to be for the agent.
-Some of the Percepts or custom actions might need some fine tuning to increase performance, but that is not really that important since a human player wouldn't respond immediately either.
-For example the Buildings percept is so large that updating it takes a good minute at the least, which, while not necessarily a problem, is quite annoying.
+Some of the Percepts (the information the agent receives from the environment) or custom actions might need some fine tuning to increase performance, but that is not really that important since a human player wouldn't respond immediately either.
+For example the Buildings percept contains so much data that updating it takes a good minute at the least, which, while not necessarily a problem, is quite annoying.
 
 \section{Evaluation of the agent}
 The agent is split up in separate modules as stated in chapter~\ref{ch:overview}.
 This really helps with keeping the code readable, because it means that we do not have all our code in the same file.
-The agent can perform almost all the actions that we wanted to implement at the start of this project.
-We left out the ability to give money to another stakeholder since we noticed that realistically nobody would just give someone else money.
+The agent can perform almost all the actions that we wanted to implement at the start of this project and work as we expected.
+This is mostly because the actions are pretty straight forward like buying ground or responding to requests made by other stakeholders.
+The only thing that we did not plan was the shape in which the agent builds the new buildings. We planned on just building in the shape of the area that the agent demolished, but in th end the buildings have a custom size which is decided by the environment. This actually looks a lot better than what we had in mind so this is a good thing.
```

Comment:
The only thing
I think there were more things, maybe safer to say "One of the things"